.github/workflows/test.yml

```
@@ -86,22 +86,39 @@ jobs:
           path: demo/log
 
   test:
-    name: Test
+    name: Test - Ruby ${{ matrix.ruby }} - Postgres ${{ matrix.pg }} - ${{ matrix.gemfile }}
```

I think Rails is likely to be the culprit in most matrix-related failures:

```suggestion
    name: Test - ${{ matrix.gemfile }} - Ruby ${{ matrix.ruby }} - PG ${{ matrix.pg }}
```